### PR TITLE
Flowmill v0.8.1067

### PR DESCRIPTION
**Pull request by juchem**
This change is [Reviewable](https://reviewable.io/reviews/flowmill/flowmill-k8s/18)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,3 @@
-## [0.8.897] - 2019-11-22
-### Changed
-    - New Agent version with multiple enhancements
-    - Intake and Services now hosted on flowmill.com
-    - Reduced number of required fields in chart
-    - Automatically create or reuse agent key secret
-
```
Comment (cirego) [inline CHANGELOG.md]: Looks like we reverted the Changelog here.

We should add a changelog entry for `1067`.

File: Chart.yaml [hunk 2]
```diff
@@ -2,5 +2,5 @@ apiVersion: v1
 appVersion: "0.7"
 description: Deploys the Flowmill agent in Kubernetes as a DaemonSet
 name: flowmill-k8s
-version: 0.8.1000
+version: 0.2.9
```
Comment (cirego) [inline Chart.yaml]: I think we should have the chart version match the agent version, since we update them in lockstep.

File: CHANGELOG.md (hunk 1 above)
Comment (juchem) [reply CHANGELOG.md]: yes, I was working on that. should be up now

related: please comment on https://flowmill.slack.com/archives/CJ41LUEGK/p1576564497014700
we need to make a call there re: changelog

File: Chart.yaml (hunk 2 above)
Comment (juchem) [reply Chart.yaml]: good point, will fix

File: Chart.yaml [hunk 3]
```diff
@@ -2,5 +2,5 @@ apiVersion: v1
 appVersion: "0.7"
```
Comment (juchem) [inline Chart.yaml]: @cirego what about `appVersion`?

Comment (cirego) [reply Chart.yaml]: This should probably be "0.8" as well, although it's unclear on the distinction between the two versions.

**Comment by juchem** [comment]
@cirego it says I'm not authorized to merge. Can you go ahead and merge it?

**Pull request merged**